SkietbaanBE/SkietbaanBE/Startup.cs

```diff
@@ -19,15 +19,12 @@ public Startup(IConfiguration configuration)
         {
             Configuration = configuration;
         }
-
         public IConfiguration Configuration { get; }
-
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc();services.AddDbContext<ModelsContext>
-                (options => options.UseSqlServer(Configuration.GetConnectionString("SkietbaanDatabase")));
-
+            services.AddMvc(); services.AddDbContext<ModelsContext>
```

Comment:
Please separate the commands `services.AddMvc();` and `services.AddDbContext<ModelsContext>` into separate lines. Will make reading the code easier and there should not be any additional code after the semi-colon